aws-predictions/src/main/java/com/amplifyframework/predictions/aws/service/RunFaceLivenessSession.kt

```diff
@@ -179,8 +179,8 @@ internal class RunFaceLivenessSession(
         }
     }
 
-    private fun stopLivenessSession() {
+    private fun stopLivenessSession(reasonCode: Int?) {
         livenessWebSocket.clientStoppedSession = true
-        livenessWebSocket.destroy()
+        livenessWebSocket.destroy(reasonCode ?: LivenessWebSocket.NORMAL_SOCKET_CLOSURE_STATUS_CODE)
```

Comment:
destroy() already has default value for parameter reasonCode= NORMAL_SOCKET_CLOSURE_STATUS_CODE.
Suggestion: This could be refactored like:
```kotlin
reasonCode?.let { livenessWebSocket.destroy(it) } ?: livenessWebSocket.destroy()
```

This will allow us to keep access modifier for NORMAL_SOCKET_CLOSURE_STATUS_CODE unchanged.